doc/releasenotes/15_0_0_release_notes.md

```diff
@@ -0,0 +1,7 @@
+# Release of Vitess v15.0.0
+The entire changelog for this release can be found [here](https:/github.com/vitessio/vitess/blob/main/doc/releasenotes/15_0_0_changelog.md).
+
+The release includes 21 commits (excluding merges)
+
+Thanks to all our contributors: @GuptaManan100, @K-Kumar-01, @ajm188, @arvind-murty, @dbussink, @frouioui, @harshit-gangal, @notfelineit, @rohit-nayak-ps, @rsajwani, @shlomi-noach, @systay
+
```

Comment:
We seem to be duplicating this in the release notes and in the changelog. I'd prefer to keep it one place. Maybe the release notes, and remove from changelog.

Reply:
Yeah, I was unsure about that, sounds good to me!